lib/coin-selection/lib/Cardano/CoinSelection/Balance.hs

```diff
@@ -1313,8 +1317,8 @@ makeChange criteria
     | otherwise =
         first mkUnableToConstructChangeError $ do
             adaAvailable <- maybeToEither
-                (requiredCost `Coin.difference` excessCoin)
-                (excessCoin `Coin.subtract` requiredCost)
+                (requiredCost <\> excessCoin)
+                (excessCoin </> requiredCost)
```

Comment:
I'm not sure I'd spot the difference very easily 🤔 Could it make it unnecessarily difficult for other people as well? I guess `subtract` and `monus` / `truncatedSubtraction` would be clearer? Or even the unicode character `∸` wikipedia [lists](https://en.wikipedia.org/wiki/Monus)... Or `<->`? (like the ledger)

Reply:
> I'm not sure I'd spot the difference very easily

Thanks for mentioning this, I do sympathise with your argument.

I'd be very happy to discuss this together with you over a call. Perhaps we can work out a better solution.

One thing to mention is that at least when _writing_ code, it's fairly hard to mix these up, since they do have very different types:

```hs
</> :: Reductive m => m -> m -> Maybe m
<\> :: Monus     m => m -> m ->       m
```

But as you point out, when _reading_ code, it can be easy to mistake one for the other.

The way I remember `<\>` is that it looks visually similar to the set subtraction operator `\`, which is indeed the monus operation for sets.

> <->? (like the ledger)

The `<->` operator in the ledger is for group subtraction, which only works for things that are actually groups. (`a <-> b` is equivalent to `a <> invert b`.) But none of the types involved in this PR are groups.

> or even the unicode character ∸

We could use the `∸` operator, as you mention, but that would then deviate from the standard operator used by [`monoid-subclasses`](https://github.com/blamario/monoid-subclasses/), so people who are used to that library would have to adjust to this new operator.

Would be happy to discuss our options together over a call!